Fix `pretraining_transforms` for `ImageEmbedder` (#1196)
PyTorchLightning/lightning-flash: CHANGELOG.md

### Fixed

> - Fixed a bug where input transforms in the `ImageEmbedder` were not called. ([1196](https://github.com/PyTorchLightning/lightning-flash/pull/1196))

Author: ```suggestion
- Fixed a bug where `pretraining_transforms` in the `ImageEmbedder` was never called. ([1196](https://github.com/PyTorchLightning/lightning-flash/pull/1196))
```
